_get_started/installation/mac.md

```diff
@@ -13,13 +13,11 @@ PyTorch is supported on macOS 10.15 (Catalina) or above.
 ### Python
 {: #mac-python}
 
-It is recommended that you use Python 3.8 - 3.11.
+It is recommended that you use Python 3.9 - 3.11.
```

Comment:
yes, I was about to mention, there is also linux.md with the same text.

Reply:
Just done already